lang/cs/Org.Apache.REEF.Client/Local/LocalClient.cs

```diff
@@ -0,0 +1,100 @@
+﻿/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+using System;
+using System.IO;
+using Org.Apache.REEF.Client.API;
+using Org.Apache.REEF.Client.Common;
+using Org.Apache.REEF.Client.Local.Parameters;
+using Org.Apache.REEF.Tang.Annotations;
+using Org.Apache.REEF.Utilities.Logging;
+
+namespace Org.Apache.REEF.Client.Local
+{
+    /// <summary>
+    ///     An implementation of the REEF interface using an external Java program
+    /// </summary>
+    public sealed class LocalClient : IREEFClient
+    {
+        /// <summary>
+        ///     The class name that contains the Java counterpart for this client.
+        /// </summary>
+        private const string JavaClassName = "org.apache.reef.bridge.client.LocalClient";
+
+        /// <summary>
+        ///     The name of the folder in the job's working directory that houses the driver.
+        /// </summary>
+        private const string DriverFolderName = "driver";
```

Comment:
Should we have org.apache.reef.runtime.common.files.REEFFileNames equivalent for c# client side ?